### DISCUSSION TITLE
The sql syntax is normal, does it show as an error?

### DISCUSSION THREAD

**Discussion by google007049** [dbeaver/dbeaver#22465] [Q&A]
### Description

![error](https://github.com/dbeaver/dbeaver/assets/58673078/ca0c1833-53c0-4460-b611-7f801c82b257)

I can't understand why this tool is called an error, even though other SQL tools have no problem with it

### DBeaver Version

community edition 23.3.2.202401072024

### Operating System

windows10

**Comment by LonwoLonwo** [comment]
Hello @google007049 

Please provide clear steps to reproduce your issue.
And query example as a text.

**Comment by google007049** [comment]
Thank you for your efforts.

**SELECT TO_DATE('20230930')  FROM dual ;** 

[Error Executing query]
The SQL above does not make an error, even if it is automatically converted by another tool. However, it does make an error in dbeaver.
Query execution failed
The SQL above is viewed as normal in dbeaver as well as other tools.



**SELECT TO_DATE('20230930','yyyy/mm/dd hh24:mi:ss')  FROM dual ;** 
[Result] 
2023-09-30 00:00:00

The SQL above is viewed as normal in dbeaver as well as other tools.


What's the difference?

I hope that normal SQL in other tools will be queried normally in Dbeaver as well.

I'm always rooting for Dbeaver. Thank you.

**Comment by LonwoLonwo** [comment]
If you want to use the TO_DATE function without [pattern parameter](https://stackoverflow.com/questions/22542882/sql-error-ora-01861-literal-does-not-match-format-string-01861) -
you need to specify the default NLS format in your connection settings then.

![2024-01-15 10_30_12-Connection _orcl_ configuration](https://github.com/dbeaver/dbeaver/assets/45152336/860662f0-717b-4aa2-b85b-e7eb66fb0820)

**Comment by google007049** [comment]
Thank you for your hard work.
When I applied it to connection settings, I can see it well.
Thank you always.
